Code/LauncherUnified/Launcher.cpp

```diff
@@ -385,24 +385,27 @@ namespace O3DELauncher
         }
         if (!projectPath.empty())
         {
-            const auto projectPathKey = FixedValueString(AZ::SettingsRegistryMergeUtils::BootstrapSettingsRootKey)
-                + "/project_path";
-            projectPathOptionOverride = FixedValueString::format(R"(--regset="%s=%.*s")",
-                projectPathKey.c_str(), aznumeric_cast<int>(projectPath.size()), projectPath.data());
-            argContainer.emplace_back(projectPathOptionOverride.data());
+            launcherJsonPatch += FixedValueString::format(R"(,
+                { "op": "add", "path": "/Amazon/AzCore/Bootstrap/project_path", "value": "%.*s" })", AZ_STRING_ARG(projectPath));
```

Comment:
Is this /Amazon for compatibility? can it be changed to O3DE?

Reply:
The setting has to start with /Amazon for backwards compatibility.
Users have set the project path using the `/Amazon/AzCore/Bootstrap/project_path` key in their own `~/.o3de/Registry/*.setreg` files and those settings still need to be read.